pkg/provisioning/service.go

```diff
@@ -89,9 +87,15 @@ func (s *Service) Init(ctx context.Context) error {
 			pipelines = append(pipelines, provPipelines...)
 		}
 	}
-	s.logger.Info(ctx).Int("count", len(pipelines)).Str("pipelinesDir", s.pipelinesPath).Str("pipelines", strings.Join(pipelines, ", ")).Msg("pipelines successfully provisioned")
 
-	s.deleteOldPipelines(ctx, pipelines)
+	deletedPls := s.deleteOldPipelines(ctx, pipelines)
+	s.logger.Info(ctx).
+		Int("created", len(pipelines)).
+		Int("deleted", len(deletedPls)).
+		Str("pipelines_path", s.pipelinesPath).
+		Strs("created_pipelines", pipelines).
+		Strs("deleted_pipelines", deletedPls).
+		Msg("pipeline configs provisioned")
```

Comment:
Nitpick / Suggestion: I'd keep it simple, people can count if they want to 😄
```suggestion
	s.logger.Info(ctx).
		Strs("created", pipelines).
		Strs("deleted", deletedPls).
		Str("pipelines_path", s.pipelinesPath).
		Msg("pipeline configs provisioned")
```